proxy/fyde_proxy_troubleshoot.md

```diff
@@ -0,0 +1,198 @@
+# Troubleshoot
+
+## Test connectivity from the device to the Fyde Access Proxy
+
+- Get the Access Proxy details from the Enterprise Console
+
+- Send a request with curl, the connection should be closed immediatelly with an empty reply
+
+  ```sh
+  → curl proxy_host:proxy_port
```

Comment:
@GMartinez-Sisti something I usually try is also
```
openssl s_client -host oporto.tech.fyde.com -port 8003
```
Tells me a TLS session opened successfully

Reply:
Nice! I'll add that.

Reply:
Updated. Please check.